### ISSUE TITLE
Services should do more...

### ISSUE THREAD

**Issue by Myoldmopar** [Myoldmopar/Cait.Rocks#58]
than just wrap an http call.   See the end of this answer: https://stackoverflow.com/a/39157243/2600110

**Comment by Myoldmopar** [comment]
At least: https://github.com/SEED-platform/seed/blob/develop/seed/static/seed/js/services/cycle_service.js#L87

**Comment by Myoldmopar** [comment]
Actually I've found resources that say they should literally just be a wrapper for the $http call and give back a promise.  It's the responsibility of the controller to determine what to do in case of an error or success, so let the controller resolve it.

**Comment by Myoldmopar** [comment]
But on the other hand, this is beautiful:

https://blog.ninja-squad.com/2015/05/28/angularjs-promises/

# Service:

```
 var getPonies = function() {
    // then() returns a new promise. We return that new promise.
    // that new promise is resolved via response.data, i.e. the ponies

    return $http.get('/api/ponies').then(function(response) {    
      return response.data;
    });
  };
```

# Controller

```
app.controller('PoniesCtrl', function($scope, ponyService) {
  ponyService.getPonies().then(function(data) {
    $scope.ponies = data;
  }).catch(function() {
    $scope.error = 'unable to get the ponies';
  });
});
```

**Comment by Myoldmopar** [comment]
Closed via #87
